website/src/scripts/setupSolidityDocs.ts

```diff
@@ -0,0 +1,71 @@
+import fs from "fs";
+import path from "path";
+
+const solidityDocDir = path.resolve(__dirname, "../../versioned_docs/version-v1.x/natspec-docs");
+const sourceDir = path.resolve(__dirname, "../../../contracts/docs");
```

Comment:
Add comment that we pull generated docs from the contracts package?

Reply:
sure